buildconfig/manylinux-build/docker_base/sdl_libs/build-sdl-libs.sh

```diff
@@ -26,6 +26,16 @@ cd ..
 # Link sdl-config into /usr/bin so that smpeg-config can find it
 ln -s /usr/local/bin/sdl-config /usr/bin/
 
+# Build smpeg.
+svn co svn://svn.icculus.org/smpeg/tags/release_0_4_5
```

Comment:
Is the `svn://` protocol secured? A quick search suggests it's not, but maybe I've missed something.

I tried to check hashes of all the tarballs downloaded because this would be a very convenient pathway for malware: Man-In-The-Middle the downloads on the build system (or hijack an expiring domain), inject malware along with a functional copy of the source code, and get it built into a popular Python package. The C sources of the dependencies aren't included in the wheel, so well-designed malware in one of the bundled libraries could be really hard to spot.

So we should be careful that anything downloaded by the build process is validated somehow to ensure it's the thing we think it is.

Reply:
Thanks. I added a hash for that too.